Dockerfile

```diff
@@ -34,6 +34,18 @@ RUN ./vgen
 RUN ./vcomp
 RUN cp obj_dir/Vswci_vtb /cheriot_ibex_safe_sim_trace
 
+FROM ubuntu:24.04 AS mpact-build
+RUN apt update && apt install -y wget git clang default-jre
+
+RUN machine=$(uname -m) \
+    && if [ "$machine" = "x86_64" ]; then bazel="linux-amd64" ; else bazel="darwin-arm64" ; fi \
```

Comment:
Why does Bazel want to pretend to be Darwin for 64-bit Arm Linux builds?

Reply:
From the CI failure, it appears that it shouldn’t: trying to run a macOS binary is failing.

Reply:
Nope. For a bit I was a little confused about the test host. The annoying thing is that I don't have access to a linux arm at work, so getting this to work has been a bit annoying..